src/lib/containers/drug_upload_tool/WarningModal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button'
+import Modal from 'react-bootstrap/Modal'
+
+export type WarningModalProps = {
+  title: string
+  copy: string | JSX.Element
+  confirmCopy?: string
+  show: boolean
+  onOK: Function
+  onCancel: Function
+  callbackArgs: any
+}
+
+export default function WarningModal(props: WarningModalProps) {
+ 
+  return (
+    <Modal show={true} animation={false}>
```

Comment:
should accept the show prop instead of passing true-- so show={props.show} and then [this line](https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/200/files#diff-cc39a698514630d6fd1a46f8127f8d0cR257) can read as `show={this.state.modalContext}`